.travis.yml

```diff
@@ -41,7 +40,7 @@ before_script:
           -DCabana_ENABLE_MPI=ON
           -DCabana_ENABLE_TESTING=OFF -DCabana_ENABLE_EXAMPLES=OFF
           -DCabana_ENABLE_PERFORMANCE_TESTING=OFF .. &&
-    make -k -j2 VERBOSE=1 && make install &&
```

Comment:
Why not the `VERBOSE=1`? This is useful for debugging output in the CI.

Reply:
Done